common/array_stack.h

```diff
@@ -67,6 +67,13 @@ class ArrayStack {
     values_.push_back(value);
   }
 
+  // Prepends a value to the top array on the stack.
+  auto PrependToTop(ValueT value) -> void {
+    CARBON_CHECK(!array_offsets_.empty(),
+                 "Must call PushArray before PushValue.");
```

Comment:
```suggestion
                 "Must call PushArray before PrependToTop.");
```
Can you also fix this in `AppendToTop` while you're here? :)

Reply:
Ah, yep - done!